src/pages/docs/security/octopus-tentacle-communication/index.md

```diff
@@ -1,23 +1,23 @@
 ---
 layout: src/layouts/Default.astro
 pubDate: 2023-01-01
-modDate: 2023-01-01
+modDate: 2025-11-14
 title: Octopus - Tentacle communication
```

Comment:
FYI: The `title` frontmatter provides the H1 for a page.

If you want the title in the navigation to differ from the on-page title, you can use:

```
  title: The title that goes on the page
  navTitle: The title that is shown in the navigation
```